src/Workspaces/Core/Portable/Rename/SymbolicRenameLocations.cs

```diff
@@ -26,7 +26,6 @@ internal sealed partial class SymbolicRenameLocations
         public readonly Solution Solution;
         public readonly ISymbol Symbol;
         public readonly SymbolRenameOptions Options;
-        public readonly CodeCleanupOptionsProvider FallbackOptions;
```

Comment:
It is removed from the RenameLocations because I think one rename symbol should map to one `SymbolicRenameLocations`, so if we have multiple rename symbols, then we should have multiple `SymbolicRenameLocations` + one CodeCleanUpOptionsProvider.